tests/ui/offset-of/offset-of-enum.rs

```diff
@@ -9,5 +9,9 @@ enum Alpha {
 
 fn main() {
     offset_of!(Alpha::One, 0); //~ ERROR expected type, found variant `Alpha::One`
-    offset_of!(Alpha, Two.0); //~ ERROR no field `Two` on type `Alpha`
+    offset_of!(Alpha, One); //~ ERROR `One` is an enum variant; expected field at end of `offset_of`
+    offset_of!(Alpha, Two.0);
+    offset_of!(Alpha, Two.1); //~ ERROR no field named `1` on enum variant `Alpha::Two`
+    offset_of!(Alpha, Two.foo); //~ ERROR no field named `foo` on enum variant `Alpha::Two`
+    offset_of!(Alpha, NonExistent); //~ ERROR no variant named `NonExistent` found for enum `Alpha`
```

Comment:
Could we also add a test for `offset_of!(Alphb, Foo)` to see the error message for an incorrect type name? (Would be good to confirm that the "did you mean ..." suggestion is offered as well)

Reply:
Also a test for just `offset_of!(Alpha)` would be good too.

Reply:
`offset_of!(Beta, One)` has been added. There are already tests for the wrong number of arguments.